.storybook/preview-head.html

```diff
@@ -0,0 +1,8 @@
+<script>
+    function applyBodyClasses() {
+        document.getElementsByTagName('body')[0].classList.add('enable-animations');
+    }
+    window.addEventListener('load', (event) => {
+        applyBodyClasses();
```

Comment:
this is apparently an anti-pattern in storybook. The addon I am using for theming can be used to set classes on the body element, so maybe we should look into this for the future.

IMO: For now this solution is fine, but we should be wary of not forgetting it. :D

Reply:
Absolutely a temporary approach for now!